users/service.go

```diff
@@ -321,12 +399,44 @@ func (svc usersService) issue(ctx context.Context, id, email string, keyType uin
 	return key.GetValue(), nil
 }
 
-func (svc usersService) identify(ctx context.Context, token string) (string, error) {
+type identityRes struct {
+	id    string
+	email string
+}
+
+func (svc usersService) identify(ctx context.Context, token string) (identityRes, error) {
 	identity, err := svc.auth.Identify(ctx, &mainflux.Token{Value: token})
 	if err != nil {
-		return "", errors.Wrap(ErrUnauthorizedAccess, err)
+		return identityRes{}, errors.Wrap(ErrUnauthorizedAccess, err)
+	}
+
+	return identityRes{identity.Id, identity.Email}, nil
+}
+
+func (svc usersService) authorize(ctx context.Context, subject, object, relation string) (bool, error) {
+	req := &mainflux.AuthorizeReq{
+		Sub: subject,
+		Obj: object,
+		Act: relation,
+	}
+	res, err := svc.auth.Authorize(ctx, req)
+	if err != nil {
+		return false, errors.Wrap(ErrAuthorization, err)
+	}
+	return res.Authorized, nil
+}
+
+func (svc usersService) addPolicy(ctx context.Context, subject, object, relation string) (bool, error) {
```

Comment:
Again - why `AddPolicy()` in this microservice, why we do not isolate adding policies in only one microservice?

Reply:
@dusanb94 opinions on this?